examples-generated/eks/cluster.yaml

```diff
@@ -0,0 +1,18 @@
+# This example manifest is auto-generated, and has not been tested.
+# Please make the necessary adjustments before using it.
+apiVersion: eks.aws.jet.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example
+spec:
+  forProvider:
+    depends_on:
```

Comment:
Are we able to remove `depends_on` sections?

Reply:
Done.